CHANGES_NEXT_RELEASE

```diff
@@ -3,4 +3,5 @@
 [cygnus-ngsi][PostGRESQLSink] Fix schemaName when inserting data into DB
 [cygnus-common][PostgreSQL, MySQL] SQL error persistance (#1791)
 [cygnus-ngsi][PostgisSink] Enable postgis functions without native types flag on.
-[cygnus-ngsi][RestHandler] Add a not null validation for ngsiVersion header of request.
\ No newline at end of file
+[cygnus-ngsi][RestHandler] Add a not null validation for ngsiVersion header of request.
+[cygnus-ngsi][HDFSSink] Separate files by property periodicity.
```

Comment:
```suggestion
[cygnus-ngsi][HDFSSink] Separate files by property periodicity (#1830)
```

Please remember to include the number of issue being fixed (in the case we have one) ;)

Reply:
Fixed in b910809